frame/support/src/weights.rs

```diff
@@ -527,30 +436,34 @@ where
 impl<Call: Encode, Extra: Encode> GetDispatchInfo for sp_runtime::testing::TestXt<Call, Extra> {
 	fn get_dispatch_info(&self) -> DispatchInfo {
 		// for testing: weight == size.
-		DispatchInfo { weight: self.encode().len() as _, pays_fee: Pays::Yes, ..Default::default() }
+		DispatchInfo {
+			weight: Weight::from_ref_time(self.encode().len() as _),
+			pays_fee: Pays::Yes,
+			..Default::default()
+		}
 	}
 }
 
 /// The weight of database operations that the runtime can invoke.
```

Comment:
Maybe we also need to document that DB weights are always calculated using computational time going forward. It's not going to be obvious whether it also includes storage weight.

Reply:
noted